core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -1249,7 +1244,7 @@ class DynamicBrokerReconfigurationTest extends QuorumTestHarness with SaslSetup
     verifyTimeout(consumerFuture)
   }
 
-  private def verifyListener(securityProtocol: SecurityProtocol, saslMechanism: Option[String], groupId: String): Unit = {
+  private def verifyListener(securityProtocol: SecurityProtocol, saslMechanism: Option[String]): Unit = {
```

Comment:
Instead of removing the group as a parameter (I understand that it is unused), please fix the validation by adding the group ID in the consumer below?

Reply:
True, we can pass `groupId` into `ConsumerBuilder`. I've pushed an update. Thanks